src/js/index.js

```diff
@@ -1,2 +1,69 @@
 import 'bootstrap/dist/css/bootstrap';
 import $ from 'jquery';
+
+import requestAdaptor from './request.js';
+
+const request = new requestAdaptor();
+
+const main = function() {
+  let data = {};
+  if (arguments[0]) data = arguments[0];
+  const showChartName = data => {
+    let name = 'Bitcoin';
+    if (data.chartName) name = data.chartName;
+    $('#chartname').html(name);
+  };
+
+  const showDate = data => {
+    let date = '';
+    if (data.time && data.time.updated) date = data.time.updated;
+    $('#date-banner-data').html(date);
+  };
+
+  const showDisclaimer = data => {
+    let disclmr = '';
+    if (data.disclaimer) disclmr = data.disclaimer;
+    $('#disclmr').html(disclmr);
```

Comment:
jQuery selector methods are resource consuming methods, so it would be recommended to cache result in a local variable for the cache.

```javascript
const $disclmr = $('#disclmr');

const showDisclaimer = data => {
  $disclmr.html(data.disclaimer || '');
}
```